src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -102,14 +107,19 @@ public Task<int> Start()
             }
         }
 
-        public void Dispose()
+        public void Stop()
         {
             lock (Lock)
             {
                 TerminateRunningProcess();
             }
         }
 
+        public void Dispose()
```

Comment:
Right now this class is MEF-exportable, so it seems odd to call dispose on it, even though it behaves the same as Stop(). 
So I added the stop but it's redundant functionality-wise. 
When we make it non-MEF (for typescript), the dispose will make sense and we could remove the Stop.
WDYT? should I just call dispose now?

Reply:
Agreed, it is odd - we should never call Dispose on a MEF-exported object. It should just be there for VS to call when it is shutting down (although I'm not sure there is a strong guarantee by the runtime that Dispose will be called when a managed process is shutting down).

I'd either leave it as MEF-exportable and make the implementation of IDisposable explicit (so Dispose isn't part of the "normal" visible API for the class), or stop exporting it. Whichever is simpler - we might end up changing it for TypeScript anyway.

Reply:
I'd leave it as-is for now, if that's ok. I'm quite sure we will change it when we do typescript, and then I'd want the dispose method public on it.